totalRP3/Modules/Register/Main/RegisterTooltip.lua

```diff
@@ -768,14 +768,38 @@ local function writeTooltipForCharacter(targetID, _, targetType)
 			---@type Player
 			local targetTarget = AddOn_TotalRP3.Player.static.CreateFromCharacterID(targetTargetID)
 			local _, targetEnglishClass = UnitClass(targetType .. "target");
-			local targetInfo = getCharacterInfoTab(targetTargetID);
-			local targetClassColor = targetEnglishClass and TRP3_API.GetClassDisplayColor(targetEnglishClass) or TRP3_API.CreateColor(1, 1, 1, 1);
+			local unitType = TRP3_API.ui.misc.getTargetType("target");
```

Comment:
This queries the type of the unit you're actively targeting, rather than the thing the tooltip is actually looking at.

Reply:
Thank you, no clue why I missed that one. Should be fixed in latest commit.